lattice_mc/simulation.py

```diff
@@ -271,23 +271,25 @@ def collective_correlation( self ):
             return None
 
     @property
-    def reduced_ionic_conductivity( self ):
+    def collective_diffusion_coefficient( self ):
         """
-        Returns the reduced ionic conductivity, \u03C3\' ( D_J * n_natoms).
+        Returns the collective or "jump" diffusion coefficient multiplied by the number of atoms,
```

Comment:
This is potentially misleading. This method returns the collective diffusion coefficient, which is equal to the "jump" diffusion coefficient multiplied by the number of atoms, D_J * N.